project-charter.md

```diff
@@ -90,12 +90,30 @@ The table below lists the high-level Executive Milestones of the project and the
 
 ## 6.1 Stakeholders (Internal and External)
 
+- Developers
+- Team Lead
+- Director/CEO
+- Investors
+- Client Company
+- General Public
+- Future External Companies
+- Phone/Computer/Tablet Manufacturers
+
 ## 6.2 Roles and Responsibilities
 
 This section describes the key roles supporting the project.
 
 | Name & Organization | Project Role | Project Responsibilities |
 | ------------------- | ------------ | ------------------------ |
+| Developers               | Project Team Member  | Engage in day-to-day activities and contribute to project |
+| Team Lead                | Project Team Member  | Ensure team stays on track, work breakdowns, work assignment and provide updates to Project Manager |
+| Project Manager          | Project Team Member  | Ensure Project Manager is familiar with requirements and provide updates to Director/CEO |
```

Comment:
possible typo? to me "ensure project manager" here might make more sense to be "ensure team lead/project team/developers/etc"

Reply:
fixed